src/main/java/de/teamlapen/vampirism/config/BalanceEntityActions.java

```diff
@@ -0,0 +1,100 @@
+package de.teamlapen.vampirism.config;
+
+import de.teamlapen.lib.lib.config.BalanceValues;
+import de.teamlapen.lib.lib.config.DefaultDouble;
+import de.teamlapen.lib.lib.config.DefaultInt;
+import de.teamlapen.vampirism.VampirismMod;
+import java.io.File;
+
+public class BalanceEntityActions extends BalanceValues {
+
+    @DefaultDouble(value = 0.3, name = "Disruption health amount", comment = "health points until action will be disrupted of max health")
```

Comment:
"Health amount which an entity has to loose to disrupt an action. Relative value to max health."?

Also maybe add min and max boundry.

Reply:
exactly